### DISCUSSION TITLE
Push Notifications

### DISCUSSION THREAD

**Discussion by jemshit** [community/community#66150] [Actions]
Could you add push notifications for Actions notifications on mobile apps. When i open android app i can see notification is there (fetched from api),  but it doesn't show push notification (sometimes push notification is more desirable than email)

**Comment by keijoraamat** [comment]
Unfortunately push notifications are only for:
- Direct mentions
- Assignments to issues or pull requests
- Requests to review a pull request
- Requests to approve a deployment

Read more: [https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#managing-your-notification-settings-with-github-mobile](https://docs.github.com/en/account-and-profile/managing-subscriptions-and-notifications-on-github/setting-up-notifications/configuring-notifications#managing-your-notification-settings-with-github-mobile)
